.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "useTabs": false,
+  "printWidth": 140,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "arrowParens": "avoid"
+}
```

Comment:
The useTabs, arrowParens and trailingComma are at default values so I don't think it's worth to add theme here.

Reply:
sure.. i just took default values from main `generator-jhipster`. I will check again and update.